code/style.css

```diff
@@ -1,4 +1,148 @@
-html {
```

Comment:
You should remember about default margin/padding of your body and HTML. the best way to reset all sneaky box behaviors you can use 
```css
* {
    margin: 0;
    padding: 0;
  }
```